packages/react-dom/src/__tests__/ReactDOMSelect-test.js

```diff
@@ -9,6 +9,12 @@
 
 'use strict';
 
+// Fix JSDOM. setAttribute is supposed to throw on things that can't be implicitly toStringed.
```

Comment:
This is a little hacky but it's making the TemporalLike objects throw without us explicitly tostringing in React which we shouldn't need to, since there can be Trusted Types involved.

Reply:
Can we unset this in an afterAll so it's just for this file? Or move it to setupEnvironment.js (or setupTests.js? I don't know what the difference is)?

Reply:
Isn't it a new document for every test suite anyway? I think we rely on that anyway.

I wanted to colocate this fix so that we can choose to fix the tests instead if we need to.